protocol/binary.go

```diff
@@ -101,6 +102,10 @@ var BinaryStreamer stream.Protocol
 //      an invalid request.
 var EnvelopeAgnosticBinary EnvelopeAgnosticProtocol
 
+// EnvelopeAgnosticBinaryStreamer is the streaming equivalent of an
+// EnvelopeAgnosticBinary.
+var EnvelopeAgnosticBinaryStreamer stream.EnvelopeAgnosticProtocol
```

Comment:
Yea, I definitely agree that declaring a TBinary makes a lot of sense, but I would prefer doing this in a separate diff since deprecating Binary and EnvelopeAgnosticBinary will require quite a few changes/renames. Does that sound good?

Reply:
Yeah, we can do it separately. I'm tried to set that up a bit in #516 if it helps.

Reply:
Thanks! We're now using binary.Default. We should be all set here.